### PR TITLE
122 get players

### PR DESCRIPTION
**Pull request by Emanalabsi**

### CODE REVIEW

**Review by ahmedirheem — APPROVED** [review]
Great job Eman 🤞

File: server/services/player.ts [hunk 1]
```diff
@@ -105,4 +106,45 @@ const playerMatchesService = async (
...
+    include: [{ model: Player, attributes: ['avatar', 'UserId'] }],
+    limit: pageSize,
+    offset: offset,
+  });
+
+  const totalPages = Math.ceil(count / pageSize);
+  const paginatedItems = players;
+
+  return {
+    status: 201,
```
Comment (Saeed99Madi) [inline server/services/player.ts:135]: 201 !! and static status !!!

File: server/routes/player.ts [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+import {
+  getPlayer,
+  getPlayers,
+  playerMatches,
+  updatePlayer,
+} from '../controllers/player';
 
 export const playerRouter: Router = Router();
 
+playerRouter.post('/:page', errorWrapper(getPlayers));
```
Comment (Saeed99Madi) [inline server/routes/player.ts:12]: get players needs to be GET Method

**Review by Saeed99Madi — CHANGES_REQUESTED** [review]

**Review by alaaahmade — APPROVED** [review]

**Review by Saeed99Madi — APPROVED** [review]
Great Work but it will be better if you made some tests

**Pull request merged**